agreement/selector.go

```diff
@@ -64,7 +64,7 @@ func membership(l LedgerReader, addr basics.Address, r basics.Round, p period, s
 	balanceRound := balanceRound(r, cparams)
 	seedRound := seedRound(r, cparams)
 
-	record, err := l.BalanceRecord(balanceRound, addr)
+	record, err := l.AccountData(balanceRound, addr)
```

Comment:
this line got me confused for few seconds, as the `AccountData` is both the name of the data structure as well as the function name. I think that `GetAccountData` is a better choice for a name. ( note that the same applies for the previous `BalanceRecord` function name )

Reply:
That makes sense. I've changed it to `Lookup` which is consistent to what `ledger.Ledger` does.